cli/azd/pkg/apphost/generate.go

```diff
@@ -148,31 +171,11 @@ func BicepTemplate(manifest *Manifest) (*memfs.FS, error) {
 	return fs, nil
 }
 
-// Inputs returns a map of fully qualified input names (as the dotted pair of resource name and input name) to information
-// about the input for every input across all resources in the manifest.
-func Inputs(manifest *Manifest) (map[string]Input, error) {
-	generator := newInfraGenerator()
-
-	if err := generator.LoadManifest(manifest); err != nil {
-		return nil, err
+func handleBicepNameQuotes(name string) string {
```

Comment:
NIT: Consider naming this something a little more direct about what it is doing: `bicepQuoteNameIfNeeded` and add a comment as to why we have this (I assume it is because if we always quote it the linters get mad.  The eternal battle between code generators and linters continues).

Reply:
Yes, I want azd to generate warning-free bicep... even if it is auto-gen. 
I don't like looking at all those many warnings after running `azd infra synth`.
It would be Ok to me if the generation would be in-memory only.  But even then, all warnings came first before an error during a bicep compilation error message.  And the more warnings, the more noise for a real error.

Reply:
> Yes, I want azd to generate warning-free bicep... even if it is auto-gen.

A noble goal.  Thanks for fighting the good fight for our users :-).